.vscode/launch.json

```diff
@@ -2,6 +2,27 @@
   "version": "0.2.0",
   "configurations": [
     {
+      "name": "Debug add new component",
```

Comment:
can you add this to the end of the file or not mess with it? I want it to default to debug test.